sdk/core/core-client/package.json

```diff
@@ -52,11 +52,11 @@
   "scripts": {
     "build:samples": "echo Obsolete",
     "build:test": "npm run clean && tshy && dev-tool run build-test",
-    "build": "npm run clean && tshy && api-extractor run --local",
+    "build": "npm run clean && tshy && dev-tool run extract-api",
```

Comment:
We can move all of core in this PR if we want?

Reply:
Either way is fine for just all of core since they should all share the same `js - core - ci` pipeline.  For every core and non-core package yes NO_CI to avoid build storm, and we need to update codegen to use this command when generating for azure-sdk-for-js repo.

Reply:
Added all of core to this PR